docs/getting_started/faq.rst

```diff
@@ -7,6 +7,111 @@ Frequently Asked Questions
 FAQs
 ----
 
+Is there a changelog or release notes?
+++++++++++++++++++++++++++++++++++++++
+
+Yes! Notes for each release are currently on GitHub here:
+
+https://github.com/python-graphblas/python-graphblas/releases
+
+How do I cite Python-graphblas?
++++++++++++++++++++++++++++++++
+
+Zenodo provides DOIs for the latest and specific versions of ``python-graphblas`` here:
+
+https://doi.org/10.5281/zenodo.7328791
+
+Follow instructions in "Cite as" in the lower right of the Zenodo webpage.
+
+Where can I find help?
+++++++++++++++++++++++
+
+If you think you found a bug, have a feature request, have a question, or have a suggestion
+for improving our `documentation <https://python-graphblas.readthedocs.io/en/latest/>`_,
+please raise an issue here:
+
+https://github.com/python-graphblas/python-graphblas/issues
+
+More open-ended discussions can be posted here:
+
+https://github.com/python-graphblas/python-graphblas/discussions
+
+You may also join our chat on Discord (in the ``#graphblas`` channel):
+
+https://discord.com/invite/vur45CbwMz
+
+and join our weekly community calls (we're friendly!):
+
+https://github.com/python-graphblas/python-graphblas/issues/247
+
+How and where are decisions made?
++++++++++++++++++++++++++++++++++
+
+We seek consensus and input from all members of the community.
+Anyone with an interest in the project can join the community, contribute to the project
+design, and participate in the decision making process. We want to hear from you!
+
+We are currently a small team, and decisions must be agreed upon by both co-creators
+`Erik Welch <https://github.com/eriknw>`_ and `Jim Kitchen <https://github.com/jim22k>`_.
+PRs must be approved by at least one core developer, and Erik and Jim must both approve
+new releases.
+
+We strive for openness, transparency, and institutional neutrality.
+Discussions typically occur in our
+`weekly community calls <https://github.com/python-graphblas/python-graphblas/issues/247>`_ and
+documented in `meeting notes <https://github.com/python-graphblas/archive/tree/main/meetings>`_
+and relevant `GitHub issues <https://github.com/python-graphblas/python-graphblas/issues>`_.
+
+As our community grows, we would welcome additional core developers, a Steering Committee,
+and other roles to help manage and direct ``python-graphblas``.
+We aspire to have (and formalize) governance such as
+`NetworkX <https://networkx.org/documentation/stable/developer/nxeps/nxep-0001.html>`_
+and `Dask <https://github.com/dask/governance/blob/main/governance.md>`_ have.
+
+What is the Code of Conduct?
+++++++++++++++++++++++++++++
+
+The Code of Conduct describes expected behavior for interactions within our community
+and by individuals when they represent the project. Our full Code of Conduct with
+instructions for how to report complaints or potential CoC violations is here:
+
+https://github.com/python-graphblas/python-graphblas/blob/main/CODE_OF_CONDUCT.md
+
+What is the deprecation policy?
```

Comment:
Note: we're also very coupled with SuiteSparse:GraphBLAS and `python-suitesparse-graphblas`. We currently don't try to support those for two years--we frequently require the most recent version of these.